src/components/common/paymentCard/index.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { FC, useContext, useState } from "react";
+import styled from "styled-components";
+import data from "../constants.json";
+import Label from "../label";
+import visa from "../../../assets/paymentCard/visa.png";
+import visa_bg from "../../../assets/paymentCard/visa_bg.png";
+import Checked from "../../../assets/paymentCard/icn_checked.png";
+import Unchecked from "../../../assets/paymentCard/icn_uncheck.png";
+import {
+  fontFamilies,
+  angles,
+  colors,
+  sizes,
+  letterSpacing,
+} from "../../../variables";
+import { PaymentCardDetailsProps } from "../interfaces";
+const PaymentCard: FC<{ cardDetails: PaymentCardDetailsProps }> = ({
+  cardDetails,
+}) => {
+  const [primaryCheck, setPrimaryCheck] = useState(cardDetails.primaryStatus);
+  return (
+    <CardContainer>
+      <TickBox onClick={() => setPrimaryCheck(!primaryCheck)}>
+        <img
```

Comment:
Create common Image component and use it in all places.

Reply:
Created common component

Reply:
created

Reply:
Still using <img> here ?